src/jolt/driver/joltcb.py

```diff
@@ -186,49 +227,83 @@ def set_power(self, val):
         :returns: (None)
         """
         raise NotImplementedError()
-    
+
     def get_voltage(self):
         """
         :returns: (0 <= float <= 80): vbias in V
         """
         b = self._send_query(CMD_GET_VOLTAGE)  # 4 bytes, -80e6 - 0
         return int.from_bytes(b, 'little', signed=True) * -1e-6
-    
+
     def set_voltage(self, val):
         """
-        :param val: (0 <= float <= 80): vbias in V
+        :param val: (lower_bound <= float <= upper_bound): vbias in V
         :returns: (None)
         """
-        if not 0 <= val <= 80:
-            logging.error("Voltage %.6f out of range 0 <= vol <= 80." % val)
+        if not 0 <= val <= self.voltage_range[1]:
+            logging.warning(f"Voltage {val:.6f} out of range 0 <= vol <= {self.voltage_range[1]}")
         # Voltage is in fact negative, but this might be confusing in the GUI?
-        # Value needs to be between -80 and 0.
+        # Value needs to be between -upper and 0.
         val = -val
         b = int(val * 1e6).to_bytes(4, 'little', signed=True)
         self._send_cmd(CMD_SET_VOLTAGE, b)
-    
+
+    def set_voltage_forced(self, target, timeout=5):
+        """
+        :param target: (0 <= float <= max_voltage): in V
```

Comment:
Need to add an explanation on what it does, and mention that it's **asynchronous**